<!DOCTYPE html>
<html><head><title>RuntimeException</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">47 references to RuntimeException</div><div class="rA">System.Management.Automation (47)</div><div class="rG" id="System.Management.Automation"><div class="rF"><div class="rN">engine\ExtendedTypeSystemException.cs (2)</div>
<a href="../engine/ExtendedTypeSystemException.cs.html#41"><b>41</b>: <i>base</i>(message, innerException)</a>
<a href="../engine/ExtendedTypeSystemException.cs.html#57"><b>57</b>: <i>base</i>(</a>
</div>
<div class="rF"><div class="rN">engine\hostifaces\LocalConnection.cs (2)</div>
<a href="../engine/hostifaces/LocalConnection.cs.html#1532"><b>1532</b>: <i>base</i>(message, innerException)</a>
<a href="../engine/hostifaces/LocalConnection.cs.html#1544"><b>1544</b>: <i>base</i>(StringUtil.Format(RunspaceStrings.ErrorLoadingModulesOnRunspaceOpen, moduleName,</a>
</div>
<div class="rF"><div class="rN">engine\lang\parserutils.cs (1)</div>
<a href="../engine/lang/parserutils.cs.html#1921"><b>1921</b>e = new <i>RuntimeException</i>(message, innerException);</a>
</div>
<div class="rF"><div class="rN">engine\lang\scriptblock.cs (1)</div>
<a href="../engine/lang/scriptblock.cs.html#1352"><b>1352</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">engine\MshSecurityException.cs (1)</div>
<a href="../engine/MshSecurityException.cs.html#77"><b>77</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">engine\NativeCommandProcessor.cs (1)</div>
<a href="../engine/NativeCommandProcessor.cs.html#2343"><b>2343</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">engine\regex.cs (1)</div>
<a href="../engine/regex.cs.html#485"><b>485</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ConnectPSSession.cs (2)</div>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#731"><b>731</b>reason = new <i>RuntimeException</i>(</a>
<a href="../engine/remoting/commands/ConnectPSSession.cs.html#737"><b>737</b>reason = new <i>RuntimeException</i>(</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\DisconnectPSSession.cs (1)</div>
<a href="../engine/remoting/commands/DisconnectPSSession.cs.html#504"><b>504</b>Exception reason = new <i>RuntimeException</i>(msg, e);</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\PushRunspaceCommand.cs (1)</div>
<a href="../engine/remoting/commands/PushRunspaceCommand.cs.html#359"><b>359</b>new <i>RuntimeException</i>(message, ex), &quot;EnterPSSessionConnectSessionFailed&quot;,</a>
</div>
<div class="rF"><div class="rN">engine\remoting\commands\ReceivePSSession.cs (1)</div>
<a href="../engine/remoting/commands/ReceivePSSession.cs.html#1026"><b>1026</b>ErrorRecord errorRecord = new ErrorRecord(new <i>RuntimeException</i>(msg, reason),</a>
</div>
<div class="rF"><div class="rN">engine\remoting\common\remotingexceptions.cs (4)</div>
<a href="../engine/remoting/common/remotingexceptions.cs.html#321"><b>321</b>: <i>base</i>(message, innerException)</a>
<a href="../engine/remoting/common/remotingexceptions.cs.html#354"><b>354</b>: <i>base</i>(PSRemotingErrorInvariants.FormatResourceString(resourceString, args), innerException)</a>
<a href="../engine/remoting/common/remotingexceptions.cs.html#423"><b>423</b>: <i>base</i>(message, innerException)</a>
<a href="../engine/remoting/common/remotingexceptions.cs.html#460"><b>460</b>: <i>base</i>(PSRemotingErrorInvariants.FormatResourceString(resourceString, args), innerException)</a>
</div>
<div class="rF"><div class="rN">engine\runtime\Operations\MiscOps.cs (1)</div>
<a href="../engine/runtime/Operations/MiscOps.cs.html#2013"><b>2013</b>exception = new <i>RuntimeException</i>(message, null);</a>
</div>
<div class="rF"><div class="rN">engine\TypeTable.cs (1)</div>
<a href="../engine/TypeTable.cs.html#1789"><b>1789</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">FormatAndOutput\common\DisplayDatabase\FormatTable.cs (1)</div>
<a href="../FormatAndOutput/common/DisplayDatabase/FormatTable.cs.html#63"><b>63</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">singleshell\config\MshSnapinLoadException.cs (2)</div>
<a href="../singleshell/config/MshSnapinLoadException.cs.html#60"><b>60</b>: <i>base</i>(message, exception)</a>
<a href="../singleshell/config/MshSnapinLoadException.cs.html#89"><b>89</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">utils\CommandDiscoveryExceptions.cs (3)</div>
<a href="../utils/CommandDiscoveryExceptions.cs.html#43"><b>43</b>: <i>base</i>(BuildMessage(commandName, resourceStr, messageArgs), innerException)</a>
<a href="../utils/CommandDiscoveryExceptions.cs.html#71"><b>71</b>public CommandNotFoundException(string message, Exception innerException) : <i>base</i>(message, innerException) { }</a>
<a href="../utils/CommandDiscoveryExceptions.cs.html#359"><b>359</b>public ScriptRequiresException(string message, Exception innerException) : <i>base</i>(message, innerException) { }</a>
</div>
<div class="rF"><div class="rN">utils\CommandProcessorExceptions.cs (2)</div>
<a href="../utils/CommandProcessorExceptions.cs.html#78"><b>78</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/CommandProcessorExceptions.cs.html#93"><b>93</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">utils\ExecutionExceptions.cs (7)</div>
<a href="../utils/ExecutionExceptions.cs.html#31"><b>31</b>: <i>base</i>(RetrieveMessage(errorRecord), RetrieveException(errorRecord))</a>
<a href="../utils/ExecutionExceptions.cs.html#56"><b>56</b>: <i>base</i>(RetrieveMessage(innerException), innerException)</a>
<a href="../utils/ExecutionExceptions.cs.html#112"><b>112</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/ExecutionExceptions.cs.html#370"><b>370</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/ExecutionExceptions.cs.html#415"><b>415</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/ExecutionExceptions.cs.html#585"><b>585</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/ExecutionExceptions.cs.html#784"><b>784</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">utils\HostInterfacesExceptions.cs (2)</div>
<a href="../utils/HostInterfacesExceptions.cs.html#55"><b>55</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/HostInterfacesExceptions.cs.html#88"><b>88</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">utils\MetadataExceptions.cs (2)</div>
<a href="../utils/MetadataExceptions.cs.html#52"><b>52</b>public MetadataException(string message, Exception innerException) : <i>base</i>(message, innerException)</a>
<a href="../utils/MetadataExceptions.cs.html#62"><b>62</b>: <i>base</i>(</a>
</div>
<div class="rF"><div class="rN">utils\ParameterBinderExceptions.cs (1)</div>
<a href="../utils/ParameterBinderExceptions.cs.html#375"><b>375</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">utils\ParserException.cs (2)</div>
<a href="../utils/ParserException.cs.html#105"><b>105</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/ParserException.cs.html#120"><b>120</b>: <i>base</i>(message, innerException)</a>
</div>
<div class="rF"><div class="rN">utils\SessionStateExceptions.cs (5)</div>
<a href="../utils/SessionStateExceptions.cs.html#64"><b>64</b>: <i>base</i>(RuntimeException.RetrieveMessage(innerException), innerException)</a>
<a href="../utils/SessionStateExceptions.cs.html#95"><b>95</b>: <i>base</i>(RuntimeException.RetrieveMessage(errorRecord),</a>
<a href="../utils/SessionStateExceptions.cs.html#119"><b>119</b>: <i>base</i>(message, innerException)</a>
<a href="../utils/SessionStateExceptions.cs.html#187"><b>187</b>: <i>base</i>(</a>
<a href="../utils/SessionStateExceptions.cs.html#456"><b>456</b>: <i>base</i>(message, innerException)</a>
</div>
</div>
</body></html>